docs/docs/installation.rst

```diff
@@ -4,78 +4,12 @@
-latest version of Unitxt from https://huggingface.co/datasets/unitxt/data/tree/main.
-
-The core of Unitxt has minimal dependencies (none beyond HuggingFace evaluate).
-Note that specific metrics or other operators may required specific dependencies, which are checked before the first time they are used.
-An error message is printed if the there are missing installed dependencies.
-
-The benefit of using the HuggingFace API approach is that you can load a Unitxt dataset, just like every other HuggingFace dataset, 
-so it can be used in preexisting code without modifications.  
-However, this incurs extra overhead when HuggingFace downloads the unitxt package and does not expose all unitxt capabilities
-(e.g., defining new datasets, metrics, templates, and more).
-
-To get the full capabilities of Unitxt, install Unitxt locally from pip:
+Install Unitxt locally from pip:
 
 .. code-block:: bash
 
   pip install unitxt
 
-
-You can then use the API:
-
-.. code-block:: python
-
-  from unitxt import load_dataset,evaluate
-  from unitxt.inference import HFPipelineBasedInferenceEngine
-
-  dataset = load_dataset('card=cards.wnli,template=templates.classification.multi_class.relation.default,max_test_instances=20')
-  test_dataset = dataset["test"]
-
-  model_name="google/flan-t5-large"
-  inference_model = HFPipelineBasedInferenceEngine(model_name=model_name, max_new_tokens=32)
-  predictions = inference_model.infer(test_dataset)
-
-  dataset_with_scores = evaluate(predictions=predictions, data=test_dataset)
-  [print(item) for item in dataset_with_scores[0]['score']['global'].items()] 
-
-
-.. warning::
-   It's important not to mix calls to the Unitxt direct APIs and the HuggingFace APIs in the same program.  Use either
-   the direct Unitxt APIs or the HuggingFace APIs to load datasets and metrics.
-
-If you get an error message like:
-
-.. code-block::
-
-   datasets_modules.datasets.unitxt--data.df049865776d8814049d4543a4068e50cda79b1558dc933047f4a41d087cc120.hf_utils.UnitxtVersionsConflictError:
-   Located installed unitxt version 1.9.0 that is older than unitxt HuggingFace dataset version 1.10.0.
-
-It means that you are loading datasets using the HuggingFace API, but you also have a local version of Unitxt
-installed, and the versions are not compatible. To fix this issue, you should choose one of the following three options:
-* Update the locally installed Unitxt
-to the Unitxt HuggingFace dataset version
-* Uninstall the local Unitxt package (in case you don't require the access to Unitxt
-direct APIs), or 
-* Change the code to load the datasets using the direct Unitxt APIs without using the HuggingFace API.
-
+The core of Unitxt has a minimal set of requirements.
+Specific metrics, inference APIs and other assets, may require additional dependencies. 
+A clear error message on what needs to be installed is provided if these dependencies are not available when the specific asset is used.
```

Comment:
A general comment, maybe worth adding `what_requirements("cards.wnli")` function?

Reply:
Yes. Could be nice.